browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -385,23 +380,26 @@ void BraveBrowserView::SetStarredState(bool is_starred) {
 
 speedreader::SpeedreaderBubbleView* BraveBrowserView::ShowSpeedreaderBubble(
     speedreader::SpeedreaderTabHelper* tab_helper,
-    bool is_enabled) {
-  speedreader::SpeedreaderBubbleView* bubble = nullptr;
-  if (is_enabled) {
-    auto* speedreader_mode_bubble = new speedreader::SpeedreaderModeBubble(
-        GetLocationBarView(), tab_helper);
-    views::BubbleDialogDelegateView::CreateBubble(speedreader_mode_bubble);
-    bubble = speedreader_mode_bubble;
-  } else {
-    auto* reader_mode_bubble =
-        new speedreader::ReaderModeBubble(GetLocationBarView(), tab_helper);
-    views::BubbleDialogDelegateView::CreateBubble(reader_mode_bubble);
-    bubble = reader_mode_bubble;
-  }
-
-  bubble->Show();
-
-  return bubble;
+    speedreader::SpeedreaderBubbleLocation location) {
+  views::View* anchor = nullptr;
+  views::BubbleBorder::Arrow arrow = views::BubbleBorder::NONE;
+  switch (location) {
+    case speedreader::SpeedreaderBubbleLocation::kLocationBar:
+      anchor = GetLocationBarView();
+      arrow = views::BubbleBorder::TOP_RIGHT;
+      break;
+    case speedreader::SpeedreaderBubbleLocation::kToolbar:
+      anchor = reader_mode_toolbar_view_->toolbar();
+      arrow = views::BubbleBorder::TOP_LEFT;
+      break;
+  }
+
+  auto* reader_mode_bubble =
+      new speedreader::ReaderModeBubble(anchor, tab_helper);
```

Comment:
I'm aware that this bubble wasn't introduced in this PR and maybe my feedback for this name would be out of scope. 

Shouldn't this class be "ReaderModeBubbleView"?